inc/agent.class.php

```diff
@@ -1095,6 +1095,7 @@ public function getAgentRunURLs() {
       $ret = [];
 
       foreach ($this->getAgentBaseURLs() as $url) {
+         array_push($ret, $url."/now");
          array_push($ret, $url."/now/".$this->fields['token']);
```

Comment:
Just a point, since b9cf48ab, adding token is no more needed on the agent side and this is just ignored.

Reply:
I had tested yesterday and with the token seems not works, that's why I added the /now without the token

Reply:
My purpose was to tell the line 1099 could be removed.
If you have a failure on that URL with the token, you still should check the agent log to see if we may have an error on agent side or if your problem would be elsewhere. From my point of view, the 2 URLs are valid for the agent so adding the URL is probably not the right fix.

Reply:
You're right, I fix it